backend/src/modules/product/product.controller.ts

```diff
@@ -93,6 +109,6 @@ export class ProductController {
   async findRelatedProducts(@Headers() headers, @Param('id') id: string) {
     const related = await this.productService.findRelatedProducts(id, headers.locale);
 
-    return new ProductResponse(related ?? []);
```

Comment:
Now endpoint returns empty array. How do we get related products?

Reply:
@MatvienkoDO ,  it’s me corrected your code, because backend did not load. Please, solve smth with it.